packs/slack/sensors/slack_sensor.py

```diff
@@ -140,51 +143,56 @@ def _handle_message(self, data):
             # Skip unsupported event
             return
 
-        # Note: We resolve user and channel information to provide more context
-        user_info = self._get_user_info(user_id=data['user'])
-        channel_info = None
-        channel_id = data.get('channel', '')
-        # Grabbing info based on the type of channel the message is in.
-        if channel_id.startswith('C'):
-            channel_info = self._get_channel_info(channel_id=channel_id)
-        elif channel_id.startswith('G'):
-            channel_info = self._get_group_info(group_id=channel_id)
-
-        if not user_info or not channel_info:
-            # Deleted user or channel
-            return
-
-        # Removes formatting from messages if enabled by the user in config
-        if self._strip_formatting:
-            text = re.sub("<http.*[|](.*)>", "\\1", data['text'])
+        if 'subtype' in data and data['subtype'] in self._message_subtype_handlers:
```

Comment:
Sorry, meant to write `=`. And it's better and nice to split it across two lines, I just did it in a single line because I was too lazy to use code block syntax :)